app/javascript/components/styles.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * ActionItemModal Multiple Components
+ *
+ * Styling for these different components
+ */
+
+export const styles = () => ({
+  statusButton: {
```

Comment:
When a status button is active, there should be some different styling to indicate that a user has selected the button.

Reply:
Material UI probably has some built in styling for a selected button.